docs/dev-guide/index.md

```
@@ -4,9 +4,7 @@ In general, we prefer simplicity. We standardize on JavaScript/TypeScript (Node.
 
 ## High level view
 
-<div style="max-width: 500px;" markdown>
-![](./high-level.d2){pad="0"}
-</div>
+![](./high-level.d2){pad="0" scale="1"}
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)